<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="29437" started-at="2020-01-08T14:21:13Z" finished-at="2020-01-08T14:21:43Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="29437" started-at="2020-01-08T14:21:13Z" finished-at="2020-01-08T14:21:43Z">
      <class name="testngtesting.testingtest">
        <test-method status="PASS" signature="launch()[pri:0, instance:testngtesting.testingtest@3ecf72fd]" name="launch" is-config="true" duration-ms="29393" started-at="2020-01-08T19:51:13Z" finished-at="2020-01-08T19:51:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="FAIL" signature="verifylogin(java.lang.String, java.lang.String)[pri:0, instance:testngtesting.testingtest@3ecf72fd]" name="verifylogin" duration-ms="8" started-at="2020-01-08T19:51:43Z" finished-at="2020-01-08T19:51:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[abisheksrinivasan@protonmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[06121995]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testngtesting.testingtest.verifylogin(testingtest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifylogin -->
      </class> <!-- testngtesting.testingtest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
